Src/IronPython/Runtime/MemoryView.cs

```diff
@@ -674,41 +767,33 @@ public sealed class MemoryView : ICodeFormattable, IWeakReferenceable, IBufferPr
             if (_buffer == null) {
                 return value._buffer == null;
             }
+            // TODO: comparing flat bytes is oversimplification; besides, no data copyimg
             return tobytes().Equals(value.tobytes());
```

Comment:
Even if it's not correct, could we change this to something a bit more efficient like:
```C#
 return _buffer.AsReadOnlySpan().SequenceEqual(value._buffer.AsReadOnlySpan());
```

Reply:
Well, it would make it even less correct. At least now we have:
```python
assert memoryview(b'axc')[::2] == memoryview(b'ayc')[::2]
```
I intend to continue working on TODOs in `MemoryView` in near future under new PRs.

Reply:
Doh, did not think about strides :disappointed: